### PR TITLE
fixed mistakes

### PR DESCRIPTION
**Pull request by SyzkoMykola**
Signed-off-by: SyzkoMykola <44019543+SyzkoMykola@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@
...
...обленої системи (тут і далі наз. **ОЗ**), у противагу до прикладної її складової, яка використовує та будує інфраструктуру навколо **ОЗ**, є визначення близькості текстових файлів (зокрема програмних кодів) у структурному та семантичному розумінні, візуалізація означеної близькості.</p><p>Інфраструктурна складова проекту використоує розроблену технологію **ОЗ** у задачі автоматичного оцінювання близькості (у означеному вище розумінні) текстових даних загальної форми (тут і далі **ІЗ**). Сценарій користування наступний: <ul><li>Надати довільну кількість текстових файлів для подальшої обробки</li><li>Отримати діаграму близькості завантажених корпусів тексту за допомогою розробленого у рамках **ОЗ** рішення,</li><li>Детально вивчити різницю між двома підозрілими на схожість корпусами тексту через веб-інтерфейс</li></ul></p><p>**ІЗ** та має бути вирішена і розгорнена у вигляді веб-сервера.</p></p>Авторам не відомий сервіс, що виконує задачі **ОЗ**, **ІЗ** разом або окремо</p><p>Задача **ОЗ** є дослідницьким напрямком, у якому state-of-the-art технології продовжують з'являтися. Тому інтегрування таких технологій у прикладні рішення має цінність для стимулювання розвитку цієї галузі.</p>
```
Comment (gevorgyana) [inline README.md:7]: Тупой диффчекер на гитхабе не нашел изменения :facepalm:

**Review by gevorgyana — APPROVED** [review]

**Comment by gevorgyana** [comment]
I suggest that we rename this PR to sth like this 'Improve orthography and punctuation'

File: README.md (hunk 1 above)
Comment (Pilipets) [reply README.md:7]: ...

**Review by Pilipets — DISMISSED** [review]

**Review by gevorgyana — APPROVED** [review]
Still need a valid email.

**Review by gevorgyana — APPROVED** [review]

**Comment by gevorgyana** [comment]
@SyzkoMykola you probably forgot to commit your changes. Anyway I think this is what you meant to do, an I right?

**Review by gevorgyana — APPROVED** [review]

**Review by Pilipets — APPROVED** [review]

**Pull request merged**